test/e2e-common.sh

```diff
@@ -260,6 +266,11 @@ function install_knative_serving_standard() {
     build_knative_from_source
   fi
 
+  if (( INSTALL_ISTIO )); then
+    echo ">> Installing Istio"
+    install_istio
+  fi
```

Comment:
We need Istio to stop being special.  Perhaps we could instead:
1. Install the Serving CRDs
2. Install the networking stuff
3. Install Serving

Reply:
Sure, updated to handle istio without special. So, install CRD and namespace first.